src/Agent.Sdk/TaskPlugin.cs

```diff
@@ -163,6 +163,13 @@ public void Warning(string message)
             Output($"##vso[task.logissue type=warning;]{Escape(message)}");
         }
 
+        public void PublishTelemetry(string area, string feature, Dictionary<string, string> properties)
+        {
+            string propertiesAsJson = JsonConvert.SerializeObject(properties, Formatting.None);
```

Comment:
try using `StringUtil.ToString()` or some method from `IOUtil` instead of the raw `JsonConvert`.

Reply:
Enhanced `StringUtil.ConvertToJson()` with `Formatting` overload.